src/main/java/frc/trigon/robot/constants/FieldConstants.java

```diff
@@ -22,7 +22,7 @@ public class FieldConstants {
     public static final MirrorableTranslation3d
             SPEAKER_TRANSLATION = new MirrorableTranslation3d(TAG_ID_TO_POSE.get(SPEAKER_TAG_ID).getTranslation().plus(SPEAKER_TAG_TO_SPEAKER), true),
             TARGET_DELIVERY_POSITION = new MirrorableTranslation3d(2.5, 7, 0, true);
-    public static final MirrorablePose2d IN_FRONT_OF_AMP_POSE = new MirrorablePose2d(1.842, 8.204 - 0.48, Rotation2d.fromDegrees(90), true);
+    public static final MirrorablePose2d IN_FRONT_OF_AMP_POSE = new MirrorablePose2d(1.842, 8.204 - 0.43, Rotation2d.fromDegrees(-90), true);
```

Comment:
is this fully tested in sim then?

Reply:
It is what looked right to me. It is a tiny bit away from the wall but it looks like the right amp aligner distance from the inside of the amp.